package.json

```diff
@@ -11,7 +11,11 @@
     "cors": "^2.8.5",
     "dotenv": "^16.0.0",
     "express": "^4.17.3",
+    "joi": "^17.6.0",
```

Comment:
Ooo - I'm a massive Joi fan :) Schema validation - nice would like to see _exactly_ what you're using that for...

Reply:
Hopefully to reject badly formed requests before they get to Prisma and bring the server down